Content.Client/Kudzu/KudzuVisualizerSystem.cs

```diff
@@ -0,0 +1,24 @@
+using Content.Shared.Kudzu;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Kudzu
+{
+
+    public sealed class KudzuVisualsSystem : VisualizerSystem<KudzuVisualizerComponent>
+    {
+        protected override void OnAppearanceChange(EntityUid uid, KudzuVisualizerComponent component, ref AppearanceChangeEvent args)
+        {
+
+            if (!TryComp(uid, out SpriteComponent? sprite))
+            {
+                return;
+            }
```

Comment:
rather than getting the sprite manually just check is `args.Sprite` is null and use that
```suggestion
            if (args.Sprite == null)
                return;
```